lib/browser/browser.ts

```diff
@@ -69,10 +69,10 @@ Zone.__load_patch('XHR', (global: any, Zone: ZoneType, api: _ZonePrivate) => {
   // Treat XMLHTTPRequest as a macrotask.
   patchXHR(global);
 
-  const XHR_TASK = zoneSymbol('xhrTask');
-  const XHR_SYNC = zoneSymbol('xhrSync');
-  const XHR_LISTENER = zoneSymbol('xhrListener');
-  const XHR_SCHEDULED = zoneSymbol('xhrScheduled');
+  const XHR_TASK = Zone.__symbol__('xhrTask');
```

Comment:
I just updated `util.ts` to change `const zoneSymbol = Zone.__symbol__` and leave all other files unchanged, please review.